README.md

```diff
@@ -82,6 +84,14 @@ Each supported ecosystem has its own Git repository where all related buildpacks
 Heroku buildpacks are implemented in Rust. We developed and open-sourced a framework for writing Cloud Native Buildpacks in Rust called libcnb.rs.
 You can find [the repository on GitHub](https://github.com/heroku/libcnb.rs).
 
+### Builder Images
+
+A CNB builder image is a packaged set of buildpacks, base images and a lifecycle binary that orchestrates the build process. You can find [Heroku's CNB Builder Images here](https://github.com/heroku/cnb-builder-images).
+
```

Comment:
I wonder if we should include https://github.com/heroku/base-images too?

Reply:
As a separate "Associated Project", or just as a link under "Builder Images" when referring to [base images]( https://github.com/heroku/base-images)?

Reply:
@edmorley I've just included it as a link when referring to it in the builder images section for now https://github.com/heroku/buildpacks/pull/7/commits/6426e87374c7a3383309abd0b6cb0e433b760e9f